apache-solr/README.md

```diff
@@ -0,0 +1,140 @@
+# Apache Solr Mixin
+
+The Apache Solr mixin is a set of configurable Grafana dashboards and alerts.
+
+The Apache Solr mixin contains the following dashboards:
+
+- Apache Solr cluster overview
+- Apache Solr query performance overview
+- Apache Solr resource monitoring overview
+- Apache Solr logs overview
+
+and the following alerts:
+
+- ApacheSolrZookeeperChangeInEnsembleSize
+- ApacheSolrHighCPUUsageCritical
+- ApacheSolrHighCPUUsageWarning
+- ApacheSolrHighHeapMemoryUsageCritical
+- ApacheSolrHighHeapMemoryUsageWarning
+- ApacheSolrLowCacheHitRatio
+- ApacheSolrHighCoreErrors
+- ApacheSolrHighDocumentIndexing
+
+## Apache Solr Cluster Overview
+
+The Apache Solr cluster overview dashboard provides details on cluster, shard, replica and Zookeeper health as well as top core and error metrics.
+
+![Apache Solr Cluster Overview Dashboard 1](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-cluster-1.png)
+![Apache Solr Cluster Overview Dashboard 2](https://storage.googleapis.com/grafanalabs-integration-assets/apache-solr/screenshots/apache-solr-cluster-2.png)
+
+## Apache Solr Query Performance Overview
```

Comment:
Another nitpick here, and I've touched on it in the past, but we use "overview" far too frequently.

In this case I would consider it as follows:
- Cluster overview, because it's an overview of the various instances
- Query Performance, **without** overview added since it's a drill down dashboard
- Resource Monitoring, **without** overview since while it does give you an overview you can't really drill down further than this. 

@rgeyer what do you think?

Reply:
removed overview on Query performance and Resource monitoring dashboards